implementations/micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalfxTimer.java

```diff
@@ -98,7 +98,8 @@ public HistogramSnapshot takeSnapshot() {
                 currentSnapshot.total(), // Already delta in sfx implementation
                 currentSnapshot.max(), // Max cannot be calculated as delta, keep the
                                        // current.
-                null, // No percentile values
+                currentSnapshot.percentileValues(), // No changes to the percentile
```

Comment:
This was a bug, which was fixed in DistributionSummary, but now we have tests for this as well to ensure no future problems.